src/components/Nav/Login/Login.js

```diff
@@ -17,51 +17,55 @@ const Login = ({ modalState, closeModal }) => {
 
   const disabled = !(userData.id.length * userData.pw.length > 1);
 
-  if (!modalState) return null;
-
-  return (
-    <>
-      <Overlay onClick={closeModal} />
-      <LoginArea disabled={disabled}>
-        <LoginTop>
-          <span />
-          <button onClick={closeModal}>X</button>
-        </LoginTop>
-        <LogoArea>
-          <LogoCharacter src="/images/logo_character.jpg" />
-          <Logo alt="logo" src="/images/logo.png" />
-          <p>Paint Everything Around Me</p>
-        </LogoArea>
-        <KakaoArea>
-          <KakaoOauthBtn href={KAKAO_OAUTH_URL}>
-            <img
-              alt="loginButton"
-              src="/images/Login/kakao_login_large_wide.png"
-            />
-          </KakaoOauthBtn>
-          <LoginTypeTwo>
-            <Direction>계정정보 직접 입력하여 로그인하기</Direction>
-            <Input
-              type="text"
-              placeholder="카카오메일 아이디, 이메일, 전화번호"
-              name="id"
-              value={userData.id}
-              onChange={handleInput}
-            />
-            <Input
-              type="password"
-              placeholder="비밀번호"
-              name="pw"
-              value={userData.pw}
-              onChange={handleInput}
-            />
-            <Button disabled={disabled}>로그인</Button>
-          </LoginTypeTwo>
-        </KakaoArea>
-        <LoginCaution>로그인 완료시 상세정보 확인 가능</LoginCaution>
-      </LoginArea>
-    </>
-  );
+  if (!loginModalState) return null;
+  else {
+    return (
+      <>
+        <Overlay onClick={() => closeLoginModal()} />
+        <LoginArea
+          disabled={disabled}
+          onKeyUp={e => e.key === 'Escape' && closeLoginModal()}
+        >
+          <LoginTop>
+            <span />
+            <button onClick={() => closeLoginModal()}>X</button>
```

Comment:
```suggestion
            <button onClick={closeLoginModal}>X</button>
```

굳이 새로운 arrow function 만들 필요가 없겠네요!